### PR TITLE
Update docs (closes #5), add num-recycle and amber params and update cuda version on AF2 container

### LINKED ISSUE #5: Update docs and README
### Description of the bug

Update the files in the docs folder (including some use cases) and the README file

### Command used and terminal output

_No response_

### Relevant files

_No response_

### System information

_No response_

### PR DESCRIPTION
**Pull request by athbaltzis**
## PR checklist

- [X] This comment contains a description of changes (with reason).
- [X] Make sure your code lints (`nf-core lint`).
- [X] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [X] Usage Documentation in `docs/usage.md` is updated.
- [X] Output Documentation in `docs/output.md` is updated.
- [X] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: conf/modules.config [hunk 1]
```diff
@@ -52,7 +52,7 @@ if (params.mode == 'AF2') {
 if (params.mode == 'colabfold') {
     process {
         withName: 'RUN_COLABFOLD' {
-            ext.args   = params.useGPU ? '' : '--cpu'
+            ext.args   = [params.useGPU ? '' : '--cpu', params.useAmber ? '--amber' : ''].join(' ').trim()
```
Comment (JoseEspinosa) [inline conf/modules.config:55]: To  make it clear I changed the format, also I think we have to change the name of the parameters for consistency with other nf-core pipelines where the snake_case is used for parameters.
```suggestion
           ext.args = [
                 params.use_gpu ? '' : '--cpu', 
                 params.use_amber ? '--amber' : ''
           ].join(' ').trim()
```

File: modules/local/localcolabfold.nf [hunk 2]
```diff
@@ -10,26 +10,24 @@ process RUN_COLABFOLD {
     tuple val(seq_name), path(fasta)
     val model_type
     path db
+    val numRec
```
Comment (JoseEspinosa) [inline modules/local/localcolabfold.nf:13]: This module needs that the container is updated to version `v0.5`, doesn't it?

File: nextflow.config [hunk 3]
```diff
@@ -24,6 +24,8 @@ params {
 
     // colabfold specific
     model_type                 = "AlphaFold2-ptm" // {AlphaFold2-ptm,AlphaFold2-multimer-v1,AlphaFold2-multimer-v2}
+    num_recycle                = 3
+    useAmber                   = true
```
Comment (JoseEspinosa) [inline nextflow.config:28]:
```suggestion
    use_amber                   = true
```

File: nextflow_schema.json [hunk 4]
```diff
@@ -94,6 +94,16 @@
                     "default": "AlphaFold2-ptm",
                     "description": "Model preset for 'colabfold' mode",
                     "enum": ["AlphaFold2-ptm", "AlphaFold2-multimer-v1", "AlphaFold2-multimer-v2"]
+                },
+                "num_recycle": {
+                    "type": "integer",
+                    "default": 3,
+                    "description": "Number of recycles"
+                },
+                "useAmber": {
```
Comment (JoseEspinosa) [inline nextflow_schema.json:103]:
```suggestion
                "use_amber": {
```

File: README.md [hunk 5]
```diff
@@ -82,6 +82,8 @@ On release, automated continuous integration tests run the pipeline on a full-si
        --outdir <OUTDIR> \
        --mode colabfold \
        --db <DB_PATH> \
+       --num_recycle 3 \
+       --useAmber <true/false> \
```
Comment (JoseEspinosa) [inline README.md:86]:
```suggestion
       --use_amber <true/false> \
```

**Review by JoseEspinosa — CHANGES_REQUESTED** [review]
Awesome @athbaltzis ! 😄  I just suggested updating the colabfold container version to the last one, and also I think we should use `camel_case` for the pipeline parameters for consistency with the rest of nf-core pipeline, hence I changed:
* `useAmber` to `use_amber`
* `useGPU` to `use_gpu`
I suggested in the instances I found on the PR but it would be great if you could change this parameters in all the other instances.
Thanks!

File: modules/local/localcolabfold.nf (hunk 2 above)
Comment (athbaltzis) [reply modules/local/localcolabfold.nf:13]: No, the functional version is still v0.4.

**Review by JoseEspinosa — APPROVED** [review]
Thanks @athbaltzis, very nice! 😄

**Pull request merged**